conductor/Cargo.toml

```
@@ -8,6 +8,7 @@ holochain_core_types = { path = "../core_types" }
 holochain_conductor_api = { path = "../conductor_api" }
 lib3h_sodium = "=0.0.7"
 holochain_common = { path = "../common" }
+signal-hook = "0.1.9"
```

That is probs why the fmt is failing
```suggestion
signal-hook = "=0.1.9"
```